src/md_compress_adaptive_adios2.py

```diff
@@ -0,0 +1,91 @@
+import numpy as np
+import math
+
+from pprint import pprint
+from codar.oas.MDTrSampler import MDTrSampler
+import codar.utils.pyAdios as AD
+
+from mpi4py import MPI
+
+comm = MPI.COMM_WORLD
+rank = comm.Get_rank()
+size = comm.Get_size()
+
+# For SST ending, replace 'BP' with 'SST'.
+reader = AD.pyAdios('BP')
+reader.open('../src/nwchem_xyz.bp', AD.OpenMode.READ)
+
+# to get available attributes
+#attrs = reader.available_attributes()
+#pprint(attrs)
+
+# to get available variables
+avars = reader.available_variables()
+if rank == 0:
+    print('Available Variables:')
+    pprint(avars)
+
+# Q: in the variables, there are 'NATOMS', 'OFFSET' and 'TATOMS'.
```

Comment:
@bsmind, I do not know the difference between NATOMS and TATOMS 
but let's ask Huub for this...